src/edu/psu/sweng500/team8/gui/SudokuGUI.java

```diff
@@ -66,10 +80,22 @@ public void cellChanged(Cell cell, int newNumber) {
 		clearMessage();
 		this.gameBoard.clearHighlightedIncorrectCells();
 		
+		this.board = this.gameSession.getGameBoard();
+		
+		
+		if (!board.hasOpenCells()) {
```

Comment:
This is redundant -- gameIsComplete() does the same thing. Looks like old code got merged back in.